tool/tctl/common/tctl.go

```diff
@@ -27,6 +27,7 @@ import (
 	"syscall"
 
 	"github.com/gravitational/teleport/api/breaker"
+	toolcommon "github.com/gravitational/teleport/tool/common"
```

Comment:
Please regroup and reformat the imports with `-local github.com/gravitational/teleport` (in vscode, `"gopls": { "formatting.local": "github.com/gravitational/teleport" }`).

Reply:
I didn't know gopls could be configured to do that for me, thanks!